tdrs-frontend/src/components/EditProfile/EditProfile.jsx

```diff
@@ -58,15 +71,24 @@ function EditProfile() {
   const [touched, setTouched] = useState({})
 
   useEffect(() => {
-    dispatch(fetchStts())
-  }, [dispatch])
+    if (requestAccessError) {
+      dispatch(
+        setAlert({ heading: requestAccessError.message, type: ALERT_ERROR })
+      )
+    }
+    dispatch(fetchSttList())
```

Comment:
Realized that the user would not know if the request itself failed so adding in that logic and added the corresponding test below.